tcf_website/templates/reviews/review.html

```diff
@@ -36,48 +40,69 @@
-                                </p>
-                            </strong>
-                        </div>
-                        <div class="ml-4 text-muted">
-                            <i class="fas fa-thumbs-up"></i>
-                            {{ review.recommendability }}
-                        </div>
-                        <div class="ml-4 text-muted">
-                            <i class="fa fa-user"></i>
-                            {{ review.instructor_rating }}
-                        </div>
-                        <div class="ml-4 text-muted">
-                            <i class="far fa-smile-beam"></i>
-                            {{ review.enjoyability }}
-                        </div>
-                        <div class="ml-4 text-muted">
-                            <i class="fa fa-dumbbell fa-fw"></i>
-                            {{ review.difficulty }}
-                        </div>
-                        <div class="ml-4 text-muted">
-                            <i class="fa fa-hourglass-half fa-fw"></i>
-                            {{ review.hours_per_week }}
+                <div class="review-content">
+                  <p class="card-text mb-2" style="white-space: pre-line;">{{ review.text }}</p>
+                </div>
+
+                <div class="review-footer row mt-3">
+                    <!-- Review Ratings -->
+                    <div class="review-ratings col-sm-8 d-flex align-items-end">
+                        <div class="d-flex text-center flex-wrap mb-2">
+                            <div id="review-average" class="font-weight-bold">
+                                <i class="fa fa-star fa-fw"></i>
+                                {{ review.average|floatformat:2 }}
+                            </div>
+                            <div class="ml-4 text-muted">
+                                <i class="fas fa-thumbs-up"></i>
+                                {{ review.recommendability }}
+                            </div>
+                            <div class="ml-4 text-muted">
+                                <i class="fa fa-user"></i>
+                                {{ review.instructor_rating }}
+                            </div>
+                            <div class="ml-4 text-muted">
+                                <i class="far fa-smile-beam"></i>
+                                {{ review.enjoyability }}
+                            </div>
+                            <div class="ml-4 text-muted">
+                                <i class="fa fa-dumbbell fa-fw"></i>
+                                {{ review.difficulty }}
+                            </div>
+                            <div class="ml-4 text-muted">
+                                <i class="fa fa-hourglass-half fa-fw"></i>
+                                {{ review.hours_per_week }}
+                            </div>
```

Comment:
The icon formatting is a little wonky on some mobile displays
<img width="165" alt="m" src="https://user-images.githubusercontent.com/34071587/107161312-f5b9a900-6969-11eb-9897-58bfd0b62557.PNG">

Reply:
sadd I'll have to test on more screen sizes. which mobile is this?

Reply:
It just occurred to me that we might want to ensure that we do support a set of devices/browsers. (e.g. Chrome on Windows 10, Safari on iPhone 12). Maybe add that to the wiki?

Reply:
this was for some of the galaxys (like galaxy fold and s5 I believe)

Reply:
fixed!
**Galaxy S5:**
![image](https://user-images.githubusercontent.com/4724192/108610805-94043080-73a6-11eb-94fa-0e85a79d25f6.png)

**Galaxy fold (280px width):**
![image](https://user-images.githubusercontent.com/4724192/108610785-6ae3a000-73a6-11eb-89a0-956ece3ea419.png)